app/locales/languages.js

```diff
@@ -68,6 +72,41 @@ export function findUserLang(callback) {
   });
 }
 
+// This function is for checking/change current app direction
+export function languageDirectionHandler(language, navigation) {
+  // check if is not rtl and user choose rtl language
+  if (!I18nManager.isRTL && rtlLanguages.includes(language)) {
```

Comment:
I've have changes in #571 to separate device locale and user local overrides. I hope to use `i18next` as the source of truth (because active locale vs device local can be different)

In that case, can you use `i18next.dir()`

Can you base it off the ideas in that branch?

Then with Typography, we can do `const {i18n} = useTranslation()` and check `i18n.dir(currentLocale)`

Reply:
Now we are using i18next as only source of truth about current language and directions
And Typography is now updated to be more consistent 
`const getCurrentDirection = () => languages.dir();`
`const ThemedText = styled.Text`
`writing-direction: ${getCurrentDirection};`